src/language-css/embed.js

```diff
@@ -1,12 +1,14 @@
 import { hardline } from "../document/builders.js";
 import printFrontMatter from "../utils/front-matter/print.js";
 
-async function embed(path, print, textToDoc /*, options */) {
+function embed(path, print, textToDoc /*, options */) {
   const node = path.getValue();
 
   if (node.type === "front-matter") {
-    const doc = await printFrontMatter(node, textToDoc);
-    return doc ? [doc, hardline] : "";
+    return async () => {
+      const doc = await printFrontMatter(node, textToDoc);
+      return doc ? [doc, hardline] : "";
```

Comment:
Not related to these PRs, but returning `""` from here means that the usual `print` function will be called for this node. I wonder if this is intended.

Reply:
Yes, the usual `print` should be called. https://github.com/prettier/prettier/blob/765cae25522f582b71a79e7fb10417eb4612abe6/src/language-css/printer-postcss.js#L100, but better to return `undefined`/`null` as embed designed.

Reply:
BTW the core plugin should always handle all node types in `print`, even if `embed` return a fallback doc, because we have `embeddedLanguageFormatting` option to skip `embed`.